api/src/wfl/module/covid.clj

```diff
@@ -23,6 +24,61 @@
 (defn create-covid-workload!
   [tx {:keys [source sink executor] :as request}])
 
+(defn start-covid-workload!
+  [{:keys [submission-id workspace] :as workload}])
+
+(defn ^:private snapshot-error-map
+  "Return default ExceptionInfo map for snapshot-related exceptions."
+  [{:keys [executor items] :as _workload} {:keys [id snapshot_id] :as _record}]
+  {:workspace (:workspace executor)
+   :table items
+   :id id
+   :snapshot_id snapshot_id})
```

Comment:
Why rename `items` to `table` here?
Is `record` a database row?

Reply:
Sounds like this will be imminently deprecated anyway, but my best guess was that `items` stored the name of the COVID instance table.  But now realizing that the renaming makes it harder to track down any issue, so will remove the rename.

I did intend for `record` to be a database row within the COVID instance table, following @rexwangcc 's example (but that also may be deprecated).

Reply:
Indeed, this is deprecated.